Sources/Private/Experimental/ExperimentalAnimationLayer.swift

```diff
@@ -110,7 +113,14 @@ final class ExperimentalAnimationLayer: CALayer {
     let timingConfiguration: CAMediaTimingConfiguration
   }
 
-  // Configuration for the animation that is currently playing in this layer
+  private enum PlaybackState: Equatable {
+    /// The animation is playing in real-time
+    case realtime
+    /// The animation is statically displaying a specific frame
+    case specificFrame(AnimationFrameTime)
```

Comment:
nit: to me these case names are a bit confusing, I personally would find this more obvious if it was:
```suggestion
    /// The animation is playing in real-time
    case playing
    /// The animation is statically displaying a specific frame
    case paused(frame: AnimationFrameTime)
```

Reply:
Nice, much better!